charts/shoot-core/charts/csi-alicloud/templates/csi-provisioner-ss.yaml

```diff
@@ -1,4 +1,59 @@
 {{- if .Values.enabled }}
+{{- if semverCompare ">= 1.13" .Values.kubernetesVersion -}}
```

Comment:
Why do we need this `if`-`else` block around the whole statefulset?!

Reply:
same reason as previous comment